daemon/pod/container.go

```diff
@@ -888,6 +884,14 @@ func (c *Container) addToSandbox() error {
 		return err
 	}
 
+	c.streams.NewInputPipes()
+	tty := &hypervisor.TtyIO{
+		Stdin:  c.streams.Stdin(),
+		Stdout: c.streams.Stdout(),
+		Stderr: c.streams.Stderr(),
+	}
+	c.p.sandbox.Attach(tty, c.Id(), nil)
```

Comment:
Then every container will listen the tty stream even if one does not have log at all, right?

If yes, I like it.